### ISSUE TITLE
the directory of the main.py(not Kivy Launcher's main.py) not in sys.path

### ISSUE THREAD

**Issue by gottadiveintopython** [kivy/kivy-launcher#2]
Hi, 2 days ago I built a Kivy Launcher APK(python3crystax==3.5, kivy==master)and found a problem.

```python
# main.py
import sys
import os.path

print(sys.path)
print(os.path.abspath('.'))
```

I placed `main.py` to `/sdcard/kivy/test` and ran.

```text
# sys.path
.
/data/data/jp.gottadiveintopython.kivylauncher/files/app/crystax_python/stdlib.zip
/data/data/jp.gottadiveintopython.kivylauncher/files/app/crystax_python/modules
/data/data/jp.gottadiveintopython.kivylauncher/files/app/crystax_python/site-packages

# current directory
/data/data/jp.gottadiveintopython.kivylauncher/files/app
```

`/sdcard/kivy/test` not in `sys.path` so I wrote `sys.path.append(os.path.dirname(__file__))` in main.py to avoid `ImportError`.

Also need fix

https://github.com/kivy/kivy-launcher/blob/master/main.py#L36
`run_entrypoint(entrypoint, orientation=orientation)` -> `run_entrypoint(entrypoint)`

**Comment by tito** [comment]
I added a fix for the orientation, however, i don't unsderstand the import error :/ Here the path is correctly changed, but maybe because i use python2.7 ?

**Comment by tito** [comment]
Fixed in ad5c5c6

**Comment by gottadiveintopython** [comment]
I'm struggling with buildozer. (The problem is not related to Kivy-Launcher)
So I can't test it yet. But anyways  thank you.
